pkg/rke2/psp.go

```diff
@@ -311,17 +321,16 @@ func setPSPs(clx *cli.Context) func(context.Context, daemonsConfig.Control) erro
 					}
 					ns.Annotations[namespaceAnnotationGlobalUnrestricted] = cisAnnotationValue
 				}
-			}
 
-			// apply node cluster role binding regardless of whether we're in CIS mode or not
-			if _, err := cs.RbacV1().ClusterRoleBindings().Get(ctx, nodeClusterRoleBindingName, metav1.GetOptions{}); err != nil {
-				if !apierrors.IsAlreadyExists(err) {
-					tmpl := fmt.Sprintf(nodeClusterRoleBindingTemplate, globalUnrestrictedPSPName)
-					if err := deployClusterRoleBindingFromYaml(ctx, cs, tmpl); err != nil {
-						logrus.Fatalf("psp: deploy psp: %s", err.Error())
+				if _, err := cs.RbacV1().ClusterRoleBindings().Get(ctx, nodeClusterRoleBindingName, metav1.GetOptions{}); err != nil {
+					if !apierrors.IsAlreadyExists(err) {
```

Comment:
As above, this is always visited if `err != nil` because a GET call to the api-server should never return an error categorized as "already exists".